docs/tutorial/using-backup-with-syclladb.md

```diff
@@ -0,0 +1,235 @@
+# Using Backup with SycllaDB
+
+This article will show you how to deploy Vald with ScyallaDB as backup Database using Helm and run it on your kubernetes cluster.
+
+## Overview
+
+One of the Vald's feature, Vald has auto index backup feature using MySQL + Redis or Cassandra which enables disaster recovery.
```

Comment:
```suggestion
Vald allows auto index backup using MySQL + Redis or Cassandra to enable disaster recovery.<br>
```

Reply:
```suggestion
This tutorial leads you to deploy Vald with using the external database for backup.
As one of the features, Vald can auto index backup using MySQL + Redis or Cassandra to enable disaster recovery.<br>
```

It is not good begging with auto index feature